F21/megannigg/lowercase.py

```diff
@@ -0,0 +1,30 @@
+def lowercase(word, language):
+
+#Turkish or Azerbaijani uses lowercase i with out a dot    
+    if 'tr' in language or 'az' in language:
+        word = word.replace('I','ı')
+        word = word.lower()
+        
+#Irish rule: when converting uppercase vowels to lowercase, the vowel must be proceded by a dash
+    elif 'ga' in language:
```

Comment:
I don't think "in" is strong enough here; for example there are three-letter codes like "dga" for the Dagaare language spoken in Ghana.